core/src/test/scala/matryoshka/spec.scala

```diff
@@ -288,8 +291,8 @@ class MatryoshkaSpecs extends Specification with ScalaCheck with ScalazMatchers
       case _ => t.map(_.head).embed
     }
 
-  "Recursive" should {
-    "isLeaf" should {
+  "Recursive" >> {
+    "isLeaf" >> {
```

Comment:
Why the change to `>>` here. I’ve always been a fan of the BDD-style tests, like “X should Y” (despite generally loving operators).